### PR TITLE
feat: [SRM-11278]: fixing the alignment issue for Healthsource table

### PR DESCRIPTION
**Pull request by harshilbits**
#### Screenshots
Before:
New Note

After:
New Note

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by praneshg239 — DISMISSED** [review]

File: src/modules/85-cv/pages/health-source/HealthSourceTable/HealthSourceTable.tsx [hunk 1]
```diff
@@ -167,25 +168,27 @@ export default function HealthSourceTable({
       }
       return (
         <>
```
Comment (praneshg239) [inline src/modules/85-cv/pages/health-source/HealthSourceTable/HealthSourceTable.tsx:170]: This fragment can be removed

**Review by devinder-harness — DISMISSED** [review]

**Comment by harshilbits** [comment]
retrigger titlecheck

**Review by deepesh-ui — APPROVED** [review]

**Review by praneshg239 — APPROVED** [review]

File: src/modules/85-cv/pages/health-source/HealthSourceTable/HealthSourceTable.tsx (hunk 1 above)
Comment (harshilbits) [reply src/modules/85-cv/pages/health-source/HealthSourceTable/HealthSourceTable.tsx]: done.

**Review by deepesh-ui — DISMISSED** [review]

**Review by vasuki-kalluri — APPROVED** [review]

**Comment by mtran7** [comment]
Don't think we need the word "Present" as it doesn't add additional meaning. Would check w/ Docs. CC: @michaelcretzman .

image
Also if Health Sources is an entity, would be consistent w/ capitalizing both words. 

image

**Review by mtran7 — APPROVED** [review]

**Pull request merged**